spec/betagouvbot/webhook_spec.rb

```diff
@@ -2,22 +2,37 @@
 # frozen_string_literal: true
 
 RSpec.describe BetaGouvBot::Webhook do
-  describe '/compte' do
-    let(:callback)  { 'https://bob.coop' }
-    let(:user_name) { 'bob' }
-    let(:text)      { "#{user_name} #{user_name}@email.coop password" }
-    let(:params)    { { response_url: callback, user_name: user_name, text: text } }
+  let(:user_name) { 'bob' }
+  let(:token)     { 'asdf1234' }
 
+  before do
+    stub_request(:any, 'https://beta.gouv.fr/api/v1.3/authors.json')
+      .to_return(
+        headers: { 'content-type' => 'application/json; charset=utf-8' },
+        body: [id: user_name].to_json
+      )
+  end
+
+  describe 'GET /actions' do
     before do
-      stub_request(:any, callback)
-      stub_request(:any, 'https://beta.gouv.fr/api/v1.3/authors.json')
-        .to_return(
-          headers: { 'content-type' => 'application/json; charset=utf-8' },
-          body: [id: user_name].to_json
-        )
+      allow(ENV).to receive(:[])
+      allow(ENV).to receive(:[]).with('apiKey') { token }
+      allow(ENV).to receive(:[]).with('consumerKey') { token }
+      stub_request(:any, /api.ovh.com/).to_return(body: [].to_json)
     end
 
+    it { expect(get('/actions')).to be_ok }
+  end
+
+  describe 'POST /compte' do
```

Comment:
The intent of the PR is to « solve glob require problem by refactoring rules ». This test ensures nothing is broken. Without it, I'd myself reject the PR. What is it that you find secondary?